charts/backstage/templates/backstage-deployment.yaml

```diff
@@ -28,8 +28,14 @@ spec:
     metadata:
       labels: {{- include "common.labels.standard" . | nindent 8 }}
         app.kubernetes.io/component: backstage
+        {{- if .Values.commonLabels }}
```

Comment:
honestly I would use different values, otherwise we won't have any way to differentiate Deployments' labels from Pods' labels.
Maybe `backstage.podLabels` and `backstage.podAnnotations` could be good candidates?

Reply:
:thinking: Is there a specific need to distinguish between a deployment and a pod using labels? I consider these common labels to be useful in setting e.g. the owner of the resources, those Backstage specific labels, etc. across all objects in the chart, so they would have to exist for both the deployment and the pod.

I don't mind having also `backstage.podLabels` if someone has a use for them. `backstage.podAnnotations` already exists and is used below. Please let me know if they are needed, and I'll add the `podLabels` value as well.

Reply:
Added `backstage.labels` and `backstage.podLabels` to align with the already existing annotation values.

Reply:
I think the k8s plugins only look at pods' labels. Anyway this is up to each adopters to specify the values as they wish. So it's nice if we can give them full control.

> Added backstage.labels and backstage.podLabels to align with the already existing annotation values.

`backstage.commonLabel` already existed, so we won't need an extra `backstage.labels` 😄

Reply:
`backstage.labels` is just for labeling deployments. `backstage.commonLabels` are included also in the ingress, service and servicemonitor object, and now in this PR in pods.

Reply:
I really don't know if we need `backstage.labels`. I was trying to ask for use cases in a comment above. If you think there are none, I can remove it. If there are - great, let's keep it there.

Reply:
so I would try close https://github.com/backstage/charts/issues/110 without touching anything else in this PR.
One of the reason is that, adding `commonLabels` at the Pods level is considered a breaking change which we want to avoid.
So as I recap, let's just add and keep only `backstage.podLabels` at the pod level.
I've suggested the changes!

Reply:
Suggested changes added to to the commit. Please have another look @vinzscam